WearSpeakerSample/wear/src/test/java/com/example/android/wearable/speaker/SpeakerPlayerScreenTest.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.wearable.speaker
+
+import com.google.android.horologist.annotations.ExperimentalHorologistApi
+import com.google.android.horologist.compose.layout.AppScaffold
+import com.google.android.horologist.compose.layout.ResponsiveTimeText
+import com.google.android.horologist.screenshots.FixedTimeSource
+import com.google.android.horologist.screenshots.rng.WearDevice
+import com.google.android.horologist.screenshots.rng.WearScreenshotTest
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.ParameterizedRobolectricTestRunner
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
+class SpeakerPlayerScreenTest(override val device: WearDevice) : WearScreenshotTest() {
+    override val tolerance = 0.02f
+
+    @OptIn(ExperimentalHorologistApi::class)
+    @Test
+    fun speakerPlayerScreenTest() = runTest {
+        AppScaffold(
```

Comment:
If we know the target is eventually compose preview screenshots, should we make these tests just call those preview functions and nothing more?

Reply:
good point, added in https://github.com/android/wear-os-samples/pull/1122/files#diff-cfb37523cde2d905509afd2bf7e20ebef58c2bbbc55b93f6a22cf99ee59db432R36 for I had the Preview as well